src/pages/index.tsx

```diff
@@ -4,55 +4,77 @@ import React from "react";
+      <Background />
+      <div tw="max-w-4xl z-10">
         <h1 tw="text-5xl md:text-6xl font-bold mb-12">Introduction</h1>
-        {/* Need to make this mobile friendly */}
         <div
+          // I know this is the longest single line of css I've ever written
           css={[
-            `background: linear-gradient(83.84deg, rgba(149, 203, 233, 0.9) -3.34%, rgba(71, 201, 242, 0.135) 35.77%), linear-gradient(101.71deg, rgba(227, 167, 228, 0.048) 46.42%, rgba(238, 96, 215, 0.8) 94.51%), linear-gradient(149.12deg, #3308AD 13.67%, rgba(51, 8, 173, 0) 86.42%);`,
-            tw`flex relative rounded-md w-full overflow-hidden`,
+            tw`background[linear-gradient(83.84deg, rgba(149, 203, 233, 0.9) -3.34%, rgba(71, 201, 242, 0.135) 35.77%), linear-gradient(101.71deg, rgba(227, 167, 228, 0.048) 46.42%, rgba(238, 96, 215, 0.8) 94.51%), linear-gradient(149.12deg, #3308AD 13.67%, rgba(51, 8, 173, 0) 86.42%)] dark:background[linear-gradient(31.2deg, #0B4347 -25.4%, rgba(9, 36, 45, 0.15) 80.73%), linear-gradient(109.68deg, rgba(60, 0, 61, 0.01) 30.31%, #20041C 85.93%), linear-gradient(149.12deg, #1B0066 13.67%, rgba(28, 0, 108, 0) 86.42%)]`,
+            tw`flex relative rounded-md w-full overflow-hidden border`,
           ]}
         >
           <div tw="py-12 pl-12 z-10">
             <div tw="text-3xl font-bold text-white mb-4">Quickstart Guide</div>
-            <div tw="text-xl font-normal text-white mb-12 max-w-md">
+            <div tw="text-xl font-normal text-white dark:text-gray-600 mb-12 max-w-md">
               A short guide on making your first deploy on Railway in a couple
               of minutes.
             </div>
             <ButtonLink
               href="/getting-started"
-              tw="w-40 bg-pink-700 text-white text-center text-lg hover:bg-pink-500"
+              tw="w-40 bg-pink-400 text-white text-center text-lg hover:bg-pink-500"
             >
               <div tw="flex flex-row justify-center items-center font-semibold">
                 <ArrowRight tw="mr-1" />
                 Get Started
               </div>
             </ButtonLink>
           </div>
-          <div tw="absolute -bottom-8 right-0 opacity-50 z-0 lg:opacity-100">
+          <div tw="absolute -bottom-20 -right-4 opacity-25 z-0 lg:opacity-100">
             <NextImage
-              src="/images/code-terminal.svg"
+              src="/images/code terminal.svg"
               alt="Graphic of Terminal"
               tw=""
               layout="intrinsic"
-              height={350}
-              width={310}
+              height={460}
+              width={417}
               quality={100}
             />
           </div>
         </div>
 
         <div tw="mt-8 grid grid-cols-1 gap-4 md:grid-cols-2 md:gap-8">
           <OverviewLink href="reference/templates">
-            <OverviewLinkIcon>
-              <Zap size="28" tw="text-pink-800" />
+            <OverviewLinkIcon tw="bg-pink-100 border-pink-100 dark:bg-[#291839]">
```

Comment:
`dark:bg-[#291839]` are these colors not in the color palette? Looks like we're hardcoding them for all the icons here?

Reply:
Yes, in this case, the color inversion messes up the scale, so I am hard coding them in until we can get to the bottom of that weird color variable not being found in the twin class.